daemon/list.go

```diff
@@ -53,6 +53,13 @@ func (daemon *Daemon) Containers(config *ContainersConfig) ([]*types.Container,
 
 	if i, ok := psFilters["status"]; ok {
 		for _, value := range i {
+			if value != "created" &&
+				value != "restarting" &&
+				value != "running" &&
+				value != "paused" &&
+				value != "exited" {
+				return nil, errors.New("Unrecognised filter value for status")
+			}
```

Comment:
Wondering; (yeah, I shouldn't mess with code-reviewing, LOL) would it make sense to have, e.g. a `isKnownState()` function in [daemon/state.go](https://github.com/docker/docker/blob/3c132ff62d49247abed25a5686611bcac6801b23/daemon/state.go). That would keep the list of state-strings more centralised, also makes it possible to test separately and/or re-use (if needed)

Flame me if it doesn't make sense, as I'm just passing by here :smile: